pages/learn/json-schema-examples.md

```diff
@@ -92,129 +177,403 @@ title: JSON Schema examples
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "Schema for representing a health record",
+  "type": "object",
+  "required": ["patientName", "dateOfBirth", "bloodType"],
+  "properties": {
+    "patientName": {
       "type": "string"
     },
-    "givenName": {
+    "dateOfBirth": {
+      "type": "string",
+      "format": "date"
+    },
+    "bloodType": {
       "type": "string"
     },
-    "additionalName": {
+    "allergies": {
       "type": "array",
       "items": {
         "type": "string"
       }
     },
-    "honorificPrefix": {
+    "conditions": {
       "type": "array",
       "items": {
         "type": "string"
       }
     },
-    "honorificSuffix": {
+    "medications": {
       "type": "array",
       "items": {
         "type": "string"
       }
     },
-    "nickname": {
+    "emergencyContact": {
+      "$ref": "https://example.com/user-profile.schema.json"
+    }
+  }
+}
+```
+
+**Data**
+
+```json
+{
+  "patientName": "Jane Doe",
+  "dateOfBirth": "1985-02-15",
+  "bloodType": "A+",
+  "allergies": ["Pollen", "Penicillin"],
+  "conditions": ["Hypertension", "Diabetes"],
+  "medications": ["Lisinopril", "Metformin"],
+  "emergencyContact": {
+    "username": "emergencyuser",
+    "email": "emergency@example.com"
+  }
+}
+```
+
+
+## Job posting
+
+A schema representing a job posting, including properties like `title`, `company`, `location`, `description`, `employmentType`, `salary`, and `applicationDeadline`. It also uses the `$anchor` keyword for defining an anchor.
+
+```json
+{
+  "$id": "https://example.com/job-posting.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "A representation of a job posting",
+  "type": "object",
+  "required": ["title", "company", "location", "description"],
+  "properties": {
+    "title": {
       "type": "string"
     },
-    "url": {
+    "company": {
       "type": "string"
     },
-    "email": {
-      "type": "object",
-      "properties": {
-        "type": {
-          "type": "string"
-        },
-        "value": {
-          "type": "string"
-        }
-      }
-    },
-    "tel": {
-      "type": "object",
-      "properties": {
-        "type": {
-          "type": "string"
-        },
-        "value": {
-          "type": "string"
-        }
-      }
-    },
-    "adr": { "$ref": "https://example.com/address.schema.json" },
-    "geo": { "$ref": "https://example.com/geographical-location.schema.json" },
-    "tz": {
+    "location": {
       "type": "string"
     },
-    "photo": {
+    "description": {
       "type": "string"
     },
-    "logo": {
+    "employmentType": {
       "type": "string"
     },
-    "sound": {
-      "type": "string"
+    "salary": {
+      "type": "number",
+      "minimum": 0
     },
-    "bday": {
+    "applicationDeadline": {
+      "type": "string",
+      "format": "date"
+    }
+  },
+  "$anchor": "JobPosting"
```

Comment:
Having an anchor at the root isn't especially helpful.  You can do it, but why when

- internally you can just use `$ref: #` instead of `$ref: #JobPosting`
- externally, you can just `$ref: https://example.com/job-posting.schema.json` instead of `$ref: https://example.com/job-posting.schema.json#JobPosting`